pkg/api/routes.go

```diff
@@ -124,12 +129,86 @@ func (rh *RouteHandler) ListTags(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	paginate := false
+	n := -1
+
+	var err error
+
+	nQuery, ok := r.URL.Query()["n"]
+
+	if ok {
+		if len(nQuery) != 1 {
+			w.WriteHeader(http.StatusBadRequest)
+			return
+		}
+
+		var n1 int64
+
+		if n1, err = strconv.ParseInt(nQuery[0], 10, 0); err != nil {
```

Comment:
I think you can just use Atoi() if you want.

Reply:
I suppose, but staying consistent with the usage in other places in code.